common/buildings/wc_duchy_capital_buildings.txt

```diff
@@ -0,0 +1,315 @@
+﻿# SPECIAL DUCHY BUILDINGS
+# We place these at the top to make them more visible in the building interface.
+
+# REGULAR DUCHY BUILDINGS
+
+aerodrome_01 = {
+	construction_time = slow_construction_time
+
+	can_construct_potential = {
+		building_requirement_castle_city_church = { LEVEL = 01 }
+		building_requirement_civilized = yes
+		OR = {
+			scope:holder.culture = {
+				OR = {
+					has_innovation = innovation_taming_the_skies
+					has_innovation = innovation_flying_beasts
+			}
+			culture_has_light_aerial_maa = yes
+			culture_has_massive_aerial_maa = yes
+		}
+	}
+	
+	can_construct_showing_failures_only = {
+		
+		culture = {
+			has_innovation = innovation_burhs
+		}
+	}
+	
+	is_enabled = {
+		county.duchy.holder = county.holder
+	}
+	show_disabled = yes
+	
+	cost_gold = expensive_building_tier_2_cost
+	
+	character_modifier = {
+		light_aerial_damage_mult = 0.3
+		light_aerial_toughness_mult = 0.3
+		light_aerial_screen_mult = 0.3
+		light_aerial_pursuit_mult = 0.3
+		
+		massive_aerial_damage_mult = 0.3
+		massive_aerial_toughness_mult = 0.3
+		massive_aerial_screen_mult = 0.3
+		massive_aerial_pursuit_mult = 0.3
+		
+		men_at_arms_maintenance = -0.2
+	}
+	
+	next_building = aerodrome_02
+	
+	type_icon = "icon_building_aerodrome.dds"
+
+	type = duchy_capital
+	
+	ai_value = {
+		base = 20
+		modifier = {
+			factor = 2
+			scope:holder.capital_province = this
+		}
+	}
+}
+
+aerodrome_02 = {
+	construction_time = slow_construction_time
+
+	can_construct_potential = {
+		building_requirement_castle_city_church = { LEVEL = 01 }
+		building_requirement_civilized = yes
+	}
+	
+	can_construct_showing_failures_only = {
+		
+		culture = {
+			has_innovation = innovation_castle_baileys
+		}
+	}
+	
+	is_enabled = {
+		county.duchy.holder = county.holder
+	}
+	show_disabled = yes
+	
+	cost_gold = expensive_building_tier_3_cost
+	
+	character_modifier = {
+		light_aerial_damage_mult = 0.4
+		light_aerial_toughness_mult = 0.4
+		light_aerial_screen_mult = 0.4
+		light_aerial_pursuit_mult = 0.4
+		
+		massive_aerial_damage_mult = 0.4
+		massive_aerial_toughness_mult = 0.4
+		massive_aerial_screen_mult = 0.4
+		massive_aerial_pursuit_mult = 0.4
+		
+		knight_limit = 1
+		
+		men_at_arms_maintenance = -0.5
```

Comment:
Wow, it's a giant bonus. Probably need to make it more costly on the spreedsheet.

Reply:
isn't that too much anyway? -50% is a huuuuge bonus, no matter how much you have to pay for the building in advance.

Reply:
@zumbak04  Any verdict on this?

Also do you mean the knight limit modifier or the maa maintenance modifier?

Reply:
I guess we both meant `men_at_arms_maintenance`.